sdk/cosmos/azure-cosmos/azure/cosmos/_synchronized_request.py

```diff
@@ -97,11 +96,15 @@ def _Request(global_endpoint_manager, request_params, connection_policy, pipelin
             raise exceptions.CosmosClientTimeoutError()
 
     if request_params.endpoint_override:
-        base_url = request_params.endpoint_override
+        base_url = request_params.location_endpoint_to_route
```

Comment:
what is the difference between the `request_params.endpoint_override` and `request_params.location_endpoint_to_route`? 

also should change to first check `if request_params.location_endpoint_to_route` ?

Reply:
this is what they look like in a write failover scenario:
![image](https://github.com/user-attachments/assets/3a5c12b9-816e-4327-87fc-174c5ba05d8d)

`endpoint_override` works as the flag to let us know if we want to replace the endpoint with the one in location_endpoint_to_route - they also always are populated together, but I will add another check